addon/components/custom-subsidy-form-fields/estimated-cost-table/edit.js

```diff
@@ -348,13 +348,23 @@ export default class CustomSubsidyFormFieldsEstimatedCostTableEditComponent exte
       this.storeOptions.sourceGraph
     );
 
-    const validCosts = entries.filter(entry => parseInt(entry.object.value) > 0 );
-    if(!validCosts.length){
+    const inValidCosts = entries.filter(entry => isNaN(parseInt(entry.object.value)));
+    if(inValidCosts.length) {
+      this.errors.pushObject({
+        message: 'Eén van de velden werd niet correct ingevuld.'
+      });
+      this.updateTripleObject(this.estimatedCostTableSubject, validEstimatedCostTable, null);
```

Comment:
Why is it set to `null` and not `false` here ?

Reply:
Very good question, I actually considered changing it all to false but then realized that other components (see objectivesTable) also uses null so kept it for the sake of consistentie. Better to make a separate PR changing this everywhere because I am not sure if this will actually have side effects